src/main/kotlin/com/github/suusan2go/kotlinfillclass/inspections/BaseFillClassInspection.kt

```diff
@@ -172,21 +177,52 @@ class FillClassFix(
         return factory.createArgument(argumentExpression, parameter.name)
     }
 
-    private fun KotlinType.defaultValue(): String? = when {
-        KotlinBuiltIns.isBoolean(this) -> "false"
-        KotlinBuiltIns.isChar(this) -> "''"
-        KotlinBuiltIns.isDouble(this) -> "0.0"
-        KotlinBuiltIns.isFloat(this) -> "0.0f"
-        KotlinBuiltIns.isInt(this) || KotlinBuiltIns.isLong(this) || KotlinBuiltIns.isShort(this) -> "0"
-        KotlinBuiltIns.isCollectionOrNullableCollection(this) -> "arrayOf()"
-        KotlinBuiltIns.isNullableAny(this) -> "null"
-        KotlinBuiltIns.isString(this) -> "\"\""
-        KotlinBuiltIns.isListOrNullableList(this) -> "listOf()"
-        KotlinBuiltIns.isSetOrNullableSet(this) -> "setOf()"
-        KotlinBuiltIns.isMapOrNullableMap(this) -> "mapOf()"
-        this.isFunctionType -> lambdaDefaultValue()
-        this.isMarkedNullable -> "null"
-        else -> null
+    private fun ValueParameterDescriptor.fillValue(): String? {
```

Comment:
I think `ValueGenerator.getXXX` should be called from `FillDummyValuesInspection` class.

So could you change `ValueParameterDescriptor.fillValue` to protected abstract and move implementation to `FillDummyValuesInspection` and `FillEmptyValuesInspection`?